src/main/kotlin/ecommerce/model/Product.kt

```diff
@@ -0,0 +1,23 @@
+package ecommerce.model
+
+class Product(
+    val id: Long? = null,
+    var name: String = "",
+    var price: Double = 0.0,
+    var imageUrl: String = "",
+) {
+    fun update(newProduct: Product) {
+        this.name = newProduct.name
+        this.price = newProduct.price
+        this.imageUrl = newProduct.imageUrl
+    }
+
+    companion object {
+        fun toEntity(
+            product: Product,
+            id: Long,
+        ): Product {
+            return Product(id, product.name, product.price, product.imageUrl)
+        }
+    }
```

Comment:
It seems like you’re using the `Product` object as an Entity, so a method name other than `toEntity` might be more appropriate. Consider renaming it to better reflect its purpose.